### PR TITLE
arrakis par-usdc erc4626 deploy added.

### PR DESCRIPTION
**Pull request by coolnft**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/sdk/chainDeploy/mainnets/polygon.ts [hunk 1]
```diff
@@ -94,6 +95,28 @@ export const deployConfig: ChainDeployConfig = {
...
       otherParams: ["0x18DAdac6d0AAF37BaAAC811F6338427B46815a81", "10"],
     },
+    {
+      // PAR/USDC
+      strategy: "ArrakisERC4626",
+      name: "PARUSDC",
+      underlying: assets.find((a) => a.symbol === assetSymbols["arrakis_USDC_PAR_005"])!.underlying,
+      otherParams: [
+        "0x528330fF7c358FE1bAe348D23849CCed8edA5917", // arrakis mimo pool
+        "", // rewardsDestination
```
Comment (carlomazzaferro) [inline packages/sdk/chainDeploy/mainnets/polygon.ts:105]: Why is this empty?

Comment (coolnft) [reply packages/sdk/chainDeploy/mainnets/polygon.ts:105]: I don't know what would the destination but we can set the destination after contract is deployed. And I took a look at the dotdot deployed in bsc and it's giving empty there as well.

File: packages/sdk/chainDeploy/mainnets/polygon.ts [hunk 2]
```diff
@@ -94,6 +95,28 @@ export const deployConfig: ChainDeployConfig = {
...
+        ),
+      ],
+      flywheelIndices: [0],
+    },
+  ],
+  dynamicFlywheels: [
+    {
+      rewardToken: "0xADAC33f543267c4D59a8c299cF804c303BC3e4aC",
+      cycleLength: 1,
+      name: "mimo",
```
Comment (carlomazzaferro) [inline packages/sdk/chainDeploy/mainnets/polygon.ts:118]: make MIMO uppercase

File: packages/sdk/chainDeploy/mainnets/polygon.ts (hunk 1 above)
Comment (carlomazzaferro) [reply packages/sdk/chainDeploy/mainnets/polygon.ts:105]: actually, remove this config

**Review by carlomazzaferro — CHANGES_REQUESTED** [review]

**Pull request merged**